src/analysis/normed_space/finite_dimension.lean

```diff
@@ -478,3 +479,33 @@ begin
     exact norm_nonneg (g x i) },
   { exact finset.sum_nonneg (λ _ _, norm_nonneg _) }
 end
+
+lemma summable_of_is_O' {ι E F : Type*} [normed_group E] [complete_space E] [normed_group F]
+  [normed_space ℝ F] [finite_dimensional ℝ F] [complete_space F] {f : ι → E} (g : ι → F)
```

Comment:
It would probably make sense to have `g` implicit, like `f`. Same thing below.

Reply:
It's also explicit in `summable_of_is_O`. Should I change it ?

Reply:
yes, this would be better.

Reply:
Also, I don't think the completness of `F` is used here (the linter will probably complain).

Reply:
Indeed !